File: Data Structures & Algorithms/merge-sorted-array/submission-0.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        int p1=0, p2=0, k=0;
        vector<int> ans(m+n,0);
        while(p1<m && p2<n){
            if(nums1[p1]<=nums2[p2]){
                ans[k++] = nums1[p1++];
            }
            else{
                ans[k++] = nums2[p2++];
            }
        }
        while(p1<m){
            ans[k++] = nums1[p1++];
        }
        while(p2<n){
            ans[k++] = nums2[p2++];
        }
        nums1 = ans;
    }
};

File: Data Structures & Algorithms/anagram-groups/submission-0.py
class Solution:
    def groupAnagrams(self, strs: List[str]) -> List[List[str]]:
        myDict = {}
        ans = []
        idx = 0
        for string in strs:
            sortedString = ''.join(sorted(string))
            if(sortedString not in myDict):
                myDict[sortedString] = idx
                ans.append([string])
                idx += 1
            else:
                ans[myDict[sortedString]].append(string)
        return ans

            

        
        